.orchestra/config/components/toolchain/msvc.yml

```diff
@@ -172,21 +172,32 @@ toolchain/(@= name @)/vc:
         trap 'rm -f -- "$OUTPUT"' EXIT;
 
         # Suppress wine debug information
-        export WINEDEBUG=-all
+        export WINEDEBUG="-all"
+
+        # Disable .Net and mshtml
+        export WINEDLLOVERRIDES="mscoree,mshtml="
 
         # Run one dedicated mspdbsrv.exe per instance of cl.exe
-        export _MSPDBSRV_ENDPOINT_="$(python -c 'import uuid; print(str(uuid.uuid4()))')"
+        export _MSPDBSRV_ENDPOINT_="$(python3 -c 'import uuid; print(str(uuid.uuid4()))')"
         wine "$SCRIPT_DIR"/(@= bin @)/mspdbsrv.ex*e -start -spawn &
-
-        # Record mspdbsrv.exe to be killed
         MSPDBSRV_PID="$!"
-        trap 'kill "$MSPDBSRV_PID"' EXIT;
+
+        # Stop mspdbsrv.exe upon exit
+        trap 'wine "$SCRIPT_DIR"/(@= bin @)/mspdbsrv.ex*e -stop' EXIT;
+
+        # Wait for mspdbsrv.exe to be up and running
+        while ! [[ "$(readlink -f "/proc/$MSPDBSRV_PID/exe")" == *"wine"* ]]; do
+            sleep 0.2
+        done
+        sleep 0.2
```

Comment:
Is this going to work if the server is (over-)loaded? (e.g. CI running 2 builds?)

Reply:
As per our discussion, can't do much better than this easily.